### PR TITLE
Transfer To SoJ Misc UXA

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15582 &&  /bcgov/entity#15461

*Description of changes:*
* Minor misc uxa items as described in the above tickets. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrTransfers/DeathCertificate.vue [hunk 1]
```diff
@@ -26,7 +26,6 @@
           <v-col cols="3">
             <div
               class="generic-label pl-8"
-              for="death-date-time"
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:29]: Wasn't used, not a label.

File: ppr-ui/src/components/mhrTransfers/DeathCertificate.vue [hunk 2]
```diff
@@ -139,9 +138,10 @@ export default defineComponent({
     }
 
     // Validate form when prompted
-    watch(() => props.validate, (validate: boolean) => {
+    watch(() => props.validate, async (validate: boolean) => {
+      await nextTick()
       validate && (context.refs.deathCertificateForm as FormIF).validate()
-    })
+    }, { immediate: true })
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:144]: Once review confirm has been selected, we want to prompt field validation each time we open Death Cert Form .

**Review by owillborn — APPROVED** [review]
LGTM

**Comment by cameron-eyds** [comment]
> LGTM

So quick, i hadn't even posted! 🥷

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Pull request merged**